tests/index.js

```diff
@@ -1,6 +1,5 @@
-if (typeof window !== 'undefined') {
-  require('./browser')
-} else {
+require('./browser')
```

Comment:
that's a nice way to test this :+1: 

jsdom only supports really recent node versions. we should probably check if the version is recent enough before running the browser tests with jsdom.